test/unit/vl-rich-data-field.test.html

```diff
@@ -117,23 +119,25 @@
                 }]
             };
             const field = fixture('vl-rich-data-field-fixture-nested-property-array');
-            field.renderCellValue(dataRow).should.equal('<td data-title="Foo">foobar</td>');
-            field.renderCellValue({}).should.equal('<td data-title="Foo">null</td>');
+            field.valueTemplate(dataRow).outerHTML.should.equal('<td data-title="Foo">foobar</td>');
+            field.valueTemplate({}).outerHTML.should.equal('<td data-title="Foo">null</td>');
         });
 
         test('Indien er geen selector attribuut aanwezig is, zal de template content slot gebruikt worden om de kolom te visualiseren', () => {
             const field = fixture('vl-rich-data-field-content-template-fixture');
-            field.renderCellValue().replace(/\s\s+/g, '').should.equal('<td data-title="Foo"><span>foo</span></td>');
+            field.valueTemplate().outerHTML.replace(/\s\s+/g, '').should.equal('<td data-title="Foo"><span>foo</span></td>');
         });
 
-        test('Indien er geen selector attribuut en geen template content slot aanwezig is, zal er een foutmelding komen bij het renderen', () => {
+        test('Indien er geen selector attribuut en geen template content slot aanwezig is, zal de renderer functie gebruikt worden', () => {
```

Comment:
en moet ge dan de test zoals hij was nog niet overhouden als er ook nog geen renderer functie is?

Reply:
Vroeger liep het fout en daarvoor was er een unit test voorzien.

Als er momenteel geen attribute, template of renderer functie is wordt er niets getoond maar zal het ook niet (meer) fout lopen. Eventueel kunnen we de code uitbreiden met een info/warn/error log statement, maar ik vermoed dat afnemers wel zullen zien wanneer er niets getoond wordt bij implementatie?